### PR TITLE
Add working with decorators doc and updated api-report markdown

### LINKED ISSUE #6376: docs: add 'Working without decorators' to 'Apps and Experiences\Dependency Injection'
# 🙋 Documentation Request

The FAST element defining tasks can be done without using decorators according to the documentation. So does the FAST built in dependency injection capability. It's suggested to add a 'Working without decorators' section in the corresponding chapter as well as other pure Javascript practices while working with FAST in the no-transpilation world.

### PR DESCRIPTION
**Pull request by janechu**
# Pull Request

## 📖 Description

This change adds a document on working without decorators.

### 🎫 Issues

Closes #6376

### General

- [x] I have included a change request file using `$ npm run change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://github.com/microsoft/fast/blob/master/CODE_OF_CONDUCT.md#our-standards) for this project.

### CODE REVIEW

**Review by davatron5000 — APPROVED** [review]

File: packages/web-components/fast-element/docs/context/api-report.md.api.md
Comment (radium-v) [inline packages/web-components/fast-element/docs/context/api-report.md.api.md:1]: Filename is `api-report.md.api.md`?

File: packages/web-components/fast-element/docs/di/api-report.md.api.md
Comment (radium-v) [inline packages/web-components/fast-element/docs/di/api-report.md.api.md:1]: Same filename situation

File: sites/website/src/docs/advanced/working-without-decorators.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+id: working-without-decorators
+title: Working without Decorators
+sidebar_label: Working without Decorators
+keywords:
+  - decorators
+---
+
+Most of our documented examples include the use of TypeScript decorators, however as decorators are an unimplemented feature in JavaScript, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/decorators.html) for details on our implementation, using them may not be right for your project.
```
Comment (radium-v) [inline sites/website/src/docs/advanced/working-without-decorators.md:9]:
```suggestion
Most of our documented examples include the use of TypeScript decorators. However, as decorators are an unimplemented feature in JavaScript, using them may not be right for your project. See [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/decorators.html) for details on our implementation.
```

File: sites/website/src/docs/advanced/working-without-decorators.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+FASTElement.define(MyElement);
+```
+
+```html
+<my-element count="42">
+```
+
+This accepts the same configuration options as the `@attr` so for example to bind a property name that is different from an attribute name:
```
Comment (radium-v) [inline sites/website/src/docs/advanced/working-without-decorators.md:34]:
```suggestion
The static `definition` accepts the same configuration options as the `@attr` decorator. For example, to bind a property name that is different from an attribute name:
```

**Review by radium-v — CHANGES_REQUESTED** [review]

**Review by radium-v — APPROVED** [review]

**Pull request merged**